extensions.json

```diff
@@ -896,6 +896,10 @@
       "repository": "https://github.com/mitchdenny/ecdc",
       "version": "1.4.0"
     },
+    {
+      "id": "mjbvz.vscode-markdown-image-size",
```

Comment:
The extension has a different package and publisher name in its [package.json](https://github.com/mjbvz/vscode-markdown-image-size/blob/master/package.json): `bierner.markdown-image-size`

Reply:
My bad! This should be ok now